src/App.js

```diff
@@ -95,81 +145,149 @@ class App extends Component {
     })
   }
 
-  checkFilterInput = (event) => {
-    const inputValue = event.target.value.toLowerCase();
-    const options = this.state.advancedOptions;
-    const filteredGames = this.state.games
-      .filter(game => game.game.toLowerCase().includes(inputValue))
-      .sort((a, b) => a.game.localeCompare(b.game));
-    this.closePopUp();
+  toggleAdvancedSearch = () => {
     this.setState({
-      searching: inputValue,
-      filteredGames
-    });
+      showAdvancedSearch: !this.state.showAdvancedSearch
+    })
   }
   
   setAdvancedFilter = (event) => {
     let dataId = event.target.dataset.id;
-    let { card, board } = this.state.filterOptions.type;
-    let { filterOptions } = this.state;
-    
+    let holder = this.state.filterOptions;
+    let inputValue = document.querySelector('#input-value').value
```

Comment:
The two filter methods have been merged into one, that way they will work together and setState after both filters have been applied. Here I'm grabbing both starting values.

Reply:
Great that you were able to combine this new functionality with the old!

Reply:
Good thinking Matt!